### PR TITLE
chore: remove redundant category prefix

### PR DESCRIPTION
**Pull request by aviadhahami**
### Why:
Super minor change to docs to remove redundant category prefix (probably a typo)

### What's being changed
just removing the `team.` prefix to match the other actions
image

### CODE REVIEW

**Comment by cmwilson21** [comment]
@aviadhahami Thanks so much for opening a PR! I'll get this triaged for review :zap:

**Comment by aviadhahami** [comment]
Hey @cmwilson21, np at all :) 
meanwhile I'll just randomly update the main branch when I visit this PR ;)

File: content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization.md [hunk 1]
```diff
@@ -724,7 +724,7 @@ For more information, see "[AUTOTITLE](/organizations/managing-organization-sett
 | `create` | Triggered when a new team is created.
 | `demote_maintainer` | Triggered when a user was demoted from a team maintainer to a team member. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
 | `destroy` | Triggered when a team is deleted from the organization.
-| `team.promote_maintainer` | Triggered when a user was promoted from a team member to a team maintainer. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
+| `promote_maintainer` | Triggered when a user was promoted from a team member to a team maintainer. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
```
Comment (sabrowning1) [inline content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization.md:727]:
```suggestion
| `promote_maintainer` | Triggered when a user is promoted from a team member to a team maintainer. For more information, see "[AUTOTITLE](/organizations/organizing-members-into-teams/assigning-the-team-maintainer-role-to-a-team-member)."
```

Thanks for catching this! While we're tweaking this sentence, I'm making a small change here to align the verb tense with the tense used in the other descriptions.

**Review by sabrowning1 — APPROVED** [review]
👋🏼 @aviadhahami, thanks for spotting this and opening a PR to fix it! Your changes look great; I made one small tweak to fix a pre-existing grammatical error and will approve and merge this now. 🚀 Thanks again for your help maintaining our docs! ✨

File: content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization.md (hunk 1 above)
Comment (aviadhahami) [reply content/organizations/keeping-your-organization-secure/managing-security-settings-for-your-organization/audit-log-events-for-your-organization.md]: haha
my skills don't expand to grammar 🧙‍♂️

**Pull request merged**